packages/next/build/index.ts

```diff
@@ -210,7 +211,8 @@ export default async function build(
         buildId,
         previewProps,
         config,
-        loadedEnvFiles
+        loadedEnvFiles,
+        distDir
```

Comment:
This value will be different after deploying since it's an absolute path which isn't going to be the same in the serverless environment